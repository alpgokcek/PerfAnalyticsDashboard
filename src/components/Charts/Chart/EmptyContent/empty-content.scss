@import "../../../../styles/variables.scss";

.empty-content {
  height: 200px;
  border: 1px solid $colorBlackBoxShadow;
  display: flex;
  justify-content: center;
  align-items: center;
  &__wrapper {
    display: flex;
    justify-content: center;
    align-items: center;
    flex-direction: column;
    font-size: 16px;
    &-icon {
      margin-bottom: 20px;
      width: 50px;
    }
  }
}
