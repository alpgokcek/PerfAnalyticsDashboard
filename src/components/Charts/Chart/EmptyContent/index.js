import React from "react";

import "./empty-content.scss";

export default function EmptyContent() {
  return (
    <div className="empty-content">
      <div className="empty-content__wrapper">
        <img
          className="empty-content__wrapper-icon"
          src="assets/img/no-data.png"
          alt="no-data"
        />
        <div>Not content found.</div>
      </div>
    </div>
  );
}
