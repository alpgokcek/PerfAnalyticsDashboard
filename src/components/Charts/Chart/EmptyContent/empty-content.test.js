import React from "react";
import EmptyContent from "./index";

import Enzyme, { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import EnzymeAdapter from "enzyme-adapter-react-16";

Enzyme.configure({ adapter: new EnzymeAdapter() });

const setup = (props) => {
  return shallow(<EmptyContent {...props} />);
};

describe("snapshot test for EmptyContent", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = setup();
  });
  it("should render correctly", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
