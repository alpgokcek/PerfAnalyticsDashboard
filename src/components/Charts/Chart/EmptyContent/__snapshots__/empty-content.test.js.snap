// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test for EmptyContent should render correctly 1`] = `
<div
  className="empty-content"
>
  <div
    className="empty-content__wrapper"
  >
    <img
      alt="no-data"
      className="empty-content__wrapper-icon"
      src="assets/img/no-data.png"
    />
    <div>
      Not content found.
    </div>
  </div>
</div>
`;
